frontend/test/metabase/scenarios/visualizations/table.cy.spec.js

```diff
@@ -35,9 +35,13 @@ describe("scenarios > visualizations > table", () => {
       cy.findByText("Link").click();
     });
 
+    // There is a lag caused by update of the table visualization which breaks cypress typing
+    cy.wait(1000);
```

Comment:
There needs to be SOMETHING we can wait for. Arbitrary waits are highly discouraged and an anti-pattern.
I'll take a look at this and if you don't mind, I can push directly to this PR?